### PR TITLE
Wizard recipe: DBCtoDBF-v1.0.0

### PR DESCRIPTION
**Pull request by lego-yaw**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: DBCtoDBF
* Version: v1.0.0

@staticfloat please review and merge.

### CODE REVIEW

**Comment by lego-yaw** [comment]
all checks passed but i  merge is blocked (the base branch restricts merging to authorized users), how can i resolve this

**Comment by lego-yaw** [comment]
[no block]

**Review by lego-yaw — COMMENTED** [review]
git commit

**Comment by lego-yaw** [comment]
git merge

File: D/DBCtoDBF/build_tarballs.jl
Comment (lego-yaw) [inline D/DBCtoDBF/build_tarballs.jl:1]: Completed changes by removing runs on windows

Comment (amenu-bright) [reply D/DBCtoDBF/build_tarballs.jl:1]: ok

**Review by amenu-bright — APPROVED** [review]

**Review by amenu-bright — APPROVED** [review]

**Comment by amenu-bright** [comment]
ok

**Review by amenu-bright — APPROVED** [review]

File: D/DBCtoDBF/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd DBCtoDBF/
+mkdir -p ${prefix}/share/licenses/DBCtoDBF
+cp LICENSE ${prefix}/share/licenses/
+cp LICENSE ${prefix}/share/licenses/DBCtoDBF/
+cd SRC/
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:21]:
```suggestion
cd $WORKSPACE/srcdir/DBCtoDBF/SRC
install_license ../LICENSE
```

File: D/DBCtoDBF/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+make
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:22]:
```suggestion
make -j${nproc}
```

File: D/DBCtoDBF/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+mkdir -p ${prefix}/bin/
+cp dbc2dbf  ${prefix}/bin/
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:24]:
```suggestion
install -Dvm 755 "dbc2dbf${exeext}"  -t "${bindir}"
```

File: D/DBCtoDBF/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+make test
+exit
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:26]:
```suggestion
```

File: D/DBCtoDBF/build_tarballs.jl [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; )
+]
+
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:47]:
```suggestion
platforms = supported_platforms()
```

File: D/DBCtoDBF/build_tarballs.jl [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+products = [
+    ExecutableProduct("dbc2dbf", :dbc2dbf)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"13.2.0")
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:59]: https://github.com/JuliaPackaging/Yggdrasil/blob/afe8e6d6949885f0a01f1ddfaa35033739269529/CONTRIBUTING.md#compatibility-tips

Comment (lego-yaw) [reply D/DBCtoDBF/build_tarballs.jl:37]: ok

File: D/DBCtoDBF/build_tarballs.jl (hunk 5 above)
Comment (lego-yaw) [reply D/DBCtoDBF/build_tarballs.jl]: Update build_tarballs.jl

**Review by amenu-bright — APPROVED** [review]
updated build_tarballs.jl

File: D/DBCtoDBF/build_tarballs.jl (hunk 6 above)
Comment (giordano) [reply D/DBCtoDBF/build_tarballs.jl:37]: You didn't change the version of gcc

**Review by amenu-bright — APPROVED** [review]
ok

File: D/DBCtoDBF/build_tarballs.jl (hunk 6 above)
Comment (giordano) [reply D/DBCtoDBF/build_tarballs.jl:37]: You still haven't changed this.

**Review by giordano — CHANGES_REQUESTED** [review]

**Review by amenu-bright — APPROVED** [review]
ok

**Review by amenu-bright — APPROVED** [review]
ok

**Review by amenu-bright — APPROVED** [review]
ok

**Review by amenu-bright — APPROVED** [review]

**Comment by giordano** [comment]
@amenu-bright I'm not sure what's your involvement with this PR, but could you please stop continuously accepting the PR? Thanks.

File: D/DBCtoDBF/build_tarballs.jl [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+sources = [
+    GitSource("https://github.com/lego-yaw/DBCtoDBF.git", "b5333eb8378b5fae64b5de5f827e4335bade35cf")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/DBCtoDBF/SRC
+install_license ../LICENSE
+make -j${nproc} CC=${CC}
+install -Dvm 755 "dbc2dbf${exeext}"  -t "${bindir}"
```
Comment (giordano) [inline D/DBCtoDBF/build_tarballs.jl:19]:
```suggestion
install -Dvm 755 "dbc2dbf" "${bindir}/dbc2dbf${exeext}"
```

**Review by lego-yaw — COMMENTED** [review]
update build_tarballs.jl

**Review by giordano — APPROVED** [review]
Thanks for your contribution!

**Comment by lego-yaw** [comment]
thank you for the hardwork

**Pull request merged**